releases/WD-2016-07-14.html

```diff
@@ -0,0 +1,657 @@
+<!doctype html><html lang="en">
```

Comment:
AFAIK, Bikeshed does not yet co-operate with Echidna and Travis, see: https://github.com/w3c/sensors/issues/74#issuecomment-203350830

(Ping @dontcallmedom for the latest status.)

We can remove the staged snapshot from `releases/` folder when the spec gets published on TR, and tag the right commit. It is not unusual one needs to fine-tune the boilerplate, publish date etc. so staging a snapshot has proven to be practical (could do it in its own branch as well).